_data/JanakPoojary/JanakPoojary.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Janakaraj Poojary
+institution: Sardar Vallabhbhai Patel Institute of Technology
+profile_pic: JanakPoojary.png
+quote: "Hello World!"
```

Comment:
Please follow the given [instruction](https://github.com/education/graduation#3-add-your-profile-information) by removing the quotes to maintain the same format